class-sc-wp-updates-notifier.php

```diff
@@ -248,9 +251,18 @@ public function do_update_check() {
 				$themes_updated = false; // no theme updates
 			}
 			if ( $core_updated || $plugins_updated || $themes_updated ) { // Did anything come back as need updating?
-				$message  = __( 'There are updates available for your WordPress site:', 'wp-updates-notifier' ) . "\n" . $message . "\n";
+				$message  = __( 'There are updates available for your WordPress site:', 'wp-updates-notifier' ) . ' ' . get_bloginfo() . ' @ ' . home_url() . "\n" . $message . "\n";
```

Comment:
It's a bit early, but this looks like a good time to do some escaping.  esc_html and esc_url... I see that this message can be filtered before output, so here looks like a good place.